collector/internal/telemetryapi/types.go

```diff
@@ -24,6 +24,25 @@ const (
 	PlatformInitStart EventType = Platform + ".initStart"
 	// PlatformInitRuntimeDone is used when function initialization ended.
 	PlatformInitRuntimeDone EventType = Platform + ".initRuntimeDone"
+	// PlatformInitReport is used to report of function initialization.
```

Comment:
Should there also be an entry for `PlatformExtension` for what's in [collector/receiver/telemetryapireceiver/types.go](https://github.com/solarwinds/opentelemetry-lambda/pull/41/files#diff-64884b6646bfae2414b23b292c061d38b5bb5fb823f4414b79ed56aee96ecf81)? Or not necessary?

Reply:
Sounds good!